com.unity.perception/Runtime/Randomization/Samplers/SamplerMetaData.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace UnityEngine.Perception.Randomization.Samplers
+{
+    [AttributeUsage(AttributeTargets.Struct)]
+    public class SamplerMetaData : Attribute
```

Comment:
Type names are not always consistent with display naming, so sometimes this attribute is necessary (floats are System.Single in .NET). I agree that for some parameters this could be done automatically. I'll see what I can do here.

Reply:
Done